2-mandatory/2-financial-times.js

```diff
@@ -1,20 +1,40 @@
 /*
-    Imagine you are working on the Financial Times web site! They have a list of article titles stored in an array.
+    Imagine you are working on the Financial Times web site! 
+    They have a list of article titles stored in an array.
 
-    The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
-    Implement the function below, which will return a new array containing only article titles which will fit.
+    The home page of the web site has a headline section,
+     which only has space for article titles which are 65 characters or less.
+    Implement the function below, which will return a new array
+     containing only article titles which will fit.
 */
+
```

Comment:
Looks very good 👍 
For extra practice, can you try re-writing this using the `filter` array method?